src/freertos_drivers/common/MCP2515Can.hxx

```diff
@@ -648,6 +659,7 @@ private:
 
     void (*interruptEnable_)(); /**< enable interrupt callback */
     void (*interruptDisable_)(); /**< disable interrupt callback */
+    uint32_t state_; /**< present bus state */
```

Comment:
it would probably be sufficient to allocate just a few bits for this field.

Reply:
Fixed